x-pack/dockerlogbeat/pipelinemanager/config.go

```diff
@@ -0,0 +1,114 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package pipelinemanager
+
+import (
+	"crypto/sha1"
+	"errors"
+	"reflect"
+	"sort"
+)
+
+// ContainerOutputConfig has all the options we'll expect from --log-opts
+type ContainerOutputConfig struct {
```

Comment:
Is it possible to not export this type?

Reply:
It's used by the 'parent' dockerlogbeat library.